### ISSUE TITLE
Compatibility Certification request for Eclipse Jersey 4.0.0-M1 for JDK 21 (Jakarta REST 4.0)

### ISSUE THREAD

**Issue by jansupol** [jakartaee/rest#1257]
- [x] Organization Name ("Organization") and, if applicable, URL\
  Eclipse Foundation
- [x] Product Name, Version and download URL (if applicable)\
  Eclipse Jersey 4.0.0-M1, see TCK Result Summary for the download links
- [x] Specification Name, Version and download URL\
   [Jakarta RESTful Web Services 4.0](https://jakarta.ee/specifications/restful-ws/4.0/)
- [x]  TCK Version, digital SHA-256 fingerprint and download URL
[Jakarta RESTful Web Services TCK 4.0.0](https://download.eclipse.org/ee4j/restful-ws/tck/eftl/jakarta-restful-ws-tck-4.0.0.zip), SHA-256: 9494caaf67a3e5524a5ee1d6e4b749782ca1cbc920c6451583b6249e51c4ac18
- [x] Public URL of TCK Results Summary\
  [TCK results summary](https://eclipse-ee4j.github.io/jersey/tck/eclipse-jersey-4.0.0-M1-jdk21)
- [x] Any Additional Specification Certification Requirements\
  None
- [x] Java runtime used to run the implementation\
  JDK 21.0.2
- [x] Summary of the information for the certification environment, operating system, cloud, ...\
   Linux
- [x] By checking this box I acknowledge that the Organization I represent accepts the terms of the [EFTL](https://www.eclipse.org/legal/tck.php).
- [x] By checking this box I attest that all TCK requirements have been met, including any compatibility rules.

**Comment by arjantijms** [comment]
I guess this can be closed, right?

**Comment by jamezp** [comment]
> I guess this can be closed, right?

Yes. I've added the approved label and closed as per https://github.com/jakartaee/rest/issues/1254#issuecomment-2216453214
